netlogo-gui/src/main/ide/CodeCompletionPopup.scala

```diff
@@ -38,6 +38,20 @@ case class CodeCompletionPopup() {
       suggestionDisplaylist.setLayoutOrientation(JList.VERTICAL)
       suggestionDisplaylist.setVisibleRowCount(10)
       suggestionDisplaylist.setCellRenderer(new SuggestionListRenderer(eA))
+      
+      suggestionDisplaylist.addMouseListener(new MouseListener {
+        override def mouseExited(e: MouseEvent): Unit = {}
+
+        override def mouseClicked(e: MouseEvent): Unit = {}
+
+        override def mouseEntered(e: MouseEvent): Unit = {}
+
+        override def mousePressed(e: MouseEvent): Unit = {}
+
+        override def mouseReleased(e: MouseEvent): Unit = {
```

Comment:
`mouseClicked` also happens on release, as it were, but only if the press event happened on the same element (I believe). Thus it should be safe to use. Which one is better depends on the following: suppose a user presses the mouse button on one item, then moves the pointer to another and releases. Should the event trigger for the new item? Or should nothing happen?

I'm not sure which would be be better, tbh.